### DISCUSSION TITLE
Since CEF 117, the key with character 2 (not the numeric keypad) does not work

### DISCUSSION THREAD

**Discussion by RendaP** [cefsharp/CefSharp#4667] [General]
I use the Czech keyboard, where there is a Czech character **"ě"** under the key 2, but unfortunately it is still not working. Why is that? (CefSharp WPF)

**Comment by amaitland** [comment]
Issue https://github.com/chromiumembedded/cef/issues/3624

**Reply by RendaP** [reply]
Why even in the new version of CefSharp still the Czech key "ě" - key 2 does not work in WPF?
Unfortunately, since CEF 117 there are more features that have stopped working properly.
Now I can't even run two applications in parallel, in which I have CefSharp. The question is? Can I revert to the old version 116, even though there is supposedly some security issue? Thank you for your answers.

**Reply by amaitland** [reply]
> Why even in the new version of CefSharp still the Czech key "ě" - key 2 does not work in WPF?

Likely something in `Chromium` changed. Subscribe to the bug linked above.

> Now I can't even run two applications in parallel, in which I have CefSharp

Issue https://github.com/cefsharp/CefSharp/issues/4668
